netbsd/NetBSDProcessList.c

```diff
@@ -222,6 +222,9 @@ static void NetBSDProcessList_scanProcs(NetBSDProcessList* this) {
             }
             if (proc->state != '?')
             break;
+	  } else {
```

Comment:
Indentation seems broken

Reply:
Seems that entire section had issues with indentation, should be fixed in ee27a7e